docs/git-workflow.md

```diff
@@ -74,6 +74,29 @@ $ git rebase jetpack/trunk
 $ git push -f origin update/my-changes
 ```
 
+### Updating an external contributor's PR
+
+If you're working directly with Jetpack trunk and need to update an external contributor's PR, these commands will be helpful:
+
+```sh
+## Add the external contributors forked Jetpack repository (for example git remote add octocat https://github.com/octocat/jetpack.git).
+git remote add namefortheremote https://github.com/nameofremote/jetpack.git
+
+# Ask git to bring in the changes that are currently on the forked Jetpack repo's trunk branch.
+git fetch namefortheremote
+
+## Checkout the forked repositories branch (for example git checkout fix/broken-jetpack-thing).
+git checkout name-of-remote-branch
```

Comment:
I wonder whether we should just recommend `gh pr checkout` instead of all this.

That usually sets things up so `git push` (or `git push --force-with-lease`) works without having to explicitly specify the remote name and branch too.

Reply:
Yeah that's ultimately what @jeherve suggested in the Slack conversation mentioned above - we'd had issues with it originally but that's resolved now, so I've updated these docs. Thanks :)